src/ORM/Association/Loader/SelectWithPivotLoader.php

```diff
@@ -111,7 +111,7 @@ protected function _buildQuery($options)
         }
 
         $query
-           ->where($this->junctionConditions)
+            ->where($this->junctionConditions)
```

Comment:
Shouldn't this be detected by some CS tool? I mean 3 spaces seem really weird to accept, if we have tools running here in Travis.

Reply:
That is strange, it should have been reported by phpcs.

Reply:
It probably has an issue with multiline.